content/fast/use-imagemin-to-compress-images/codelab-imagemin-webpack.md

```diff
@@ -189,9 +189,7 @@ new ImageminPlugin({
 This code tells Webpack to compress JPGs to a quality of '50' ('0' is the worst;
 '100' is the best) using the Mozjpeg plugin.
 
-<div class="aside note">
```

Comment:
I removed this because stylistically, the contents didn't seem the same as the other aside notes above.  The other aside notes were actually self-contained asides - but this one was a hypothetical question answered in the two paragraphs of text directly below it.  Before I realized that the answer was addressed in regular text beneath it, I thought it was just satirical criticism of how the plugins array worked.